### PR TITLE
Ci workflows

### PR DESCRIPTION
**Pull request by KernTHeAce**
Adding workflows for:
1. Checking commit message according regex pattern
2. Running tests and 'black' and 'pylint' checks
3. Release with tag pushing

### CODE REVIEW

File: .github/workflows/message_checker.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          # A error message which will be returned in case of an error.
+          error: "One of commit messages has an incorrect title. Please read the documentation: docs/dev/pr.md"
+          # Setting this input to true will exclude the Pull Request title from the check.
+          excludeTitle: true # optional, default is false
+          # Setting this input to true will exclude the Pull Request description from the check.
+          excludeDescription: true # optional, default is false
+          # Setting this input to true will check all Pull Request commits
+          checkAllCommitMessages: true # optional, default is false
+          # you must provide GITHUB_TOKEN to this input if checkAllCommitMessages is true
+          accessToken: ${{ secrets.GITHUB_TOKEN }} # optional, default is false
```
Comment (AlexandrZagorskiy) [inline .github/workflows/message_checker.yaml]: no newline at end of file

File: .github/workflows/push_ci.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - name: Linting with pylint
+      run: |
+        poetry run pylint sc_server_auth
+
+    - name: Unittest with coverage
+      env:
+        COVERAGE_LIMIT: 50
+      run: |
+        poetry run coverage run -m unittest discover -p "*_tests.py" -v
+        poetry run coverage report --fail-under=$COVERAGE_LIMIT -m
```
Comment (AlexandrZagorskiy) [inline .github/workflows/push_ci.yaml]: no newline at end of file

File: .github/workflows/message_checker.yaml (hunk 1 above)
Comment (KernTHeAce) [reply .github/workflows/message_checker.yaml]: Done

File: .github/workflows/push_ci.yaml (hunk 2 above)
Comment (KernTHeAce) [reply .github/workflows/push_ci.yaml]: Done

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**